docs/outlook/notifications.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Create notifications for your Outlook add-in
+description: Learn about the types of notification messages you can create for your Outlook add-in.
+ms.date: 05/27/2025
+ms.localizationpriority: medium
+---
+
+# Create notifications for your Outlook add-in
+
+Implement notification messages for your Outlook add-in to keep your users informed about important events, feedback, or errors with minimal disruption to their workflow.
+
+> [!NOTE]
+> Support for the notifications API was introduced in Mailbox requirement set 1.3. Additional features were introduced in later requirement sets. To determine if your client supports these requirement sets, see [Outlook client support](/javascript/api/requirement-sets/outlook/outlook-api-requirement-sets#outlook-client-support).
+
+## Supported Outlook surfaces and modes
+
+Notification messages are supported on messages and appointments in both read and compose modes. They are displayed above the body of the mail item.
+
+:::image type="content" source="../images/outlook-notification.png" alt-text="An insight notification displayed in an appointment in compose mode.":::
```

Comment:
This image appears very fuzzy/blurry. Can we create a better image?
![image](https://github.com/user-attachments/assets/94e7bd74-7996-4ea2-898d-2453dccffdee)

Reply:
Fixed.